app/controllers/businesses_controller.rb

```diff
@@ -8,4 +8,24 @@ def show
     @business = Business.find_by(slug: params[:slug])
     @items = @business.items
   end
+
+  def edit
+    @business = Business.find(params[:id])
+  end
+
+  def update
```

Comment:
I think we need to scope the `Business.find` according to `current_user`. Otherwise, one person could edit someone else's business just by sending the business id with the request.